upup/pkg/fi/cloudup/openstacktasks/port.go

```diff
@@ -126,12 +126,19 @@ func (s *Port) Find(context *fi.Context) (*Port, error) {
 	if err != nil {
 		return nil, err
 	}
-	if rs == nil {
+	filteredList := []ports.Port{}
```

Comment:
Tip: I used to do this also (and probably still do in places), I've since learned that because a nil slice behaves (almost) identically to an empty slice, you can normally just do the shorter `var filteredList []ports.Port`.